test/onnx/expect/TestOperators.test_full_like.expect

```diff
@@ -1,6 +1,6 @@
 ir_version: 6
 producer_name: "pytorch"
-producer_version: "XXX"
+producer_version: "1.7"
```

Comment:
You don't need to include this file if the only change is the producer_version change from xxx to a version.
This allows these tests to compare model files ignoring of the producer_version.